html/partials/compare/unmanaged_files.html.haml

```diff
@@ -0,0 +1,59 @@
+- if @diff["unmanaged_files"]
+  %a.scope_anchor{ id: "unmanaged_files" }
+  #unmanaged_files_container.scope
```

Comment:
Classes should be listed before IDs (.scope should precede #unmanaged_files_container)